src/Impostor.Server/Net/Inner/Objects/InnerPlayerControl.cs

```diff
@@ -248,6 +249,12 @@ public override async ValueTask HandleRpc(ClientPlayer sender, ClientPlayer? tar
                     }
 
                     var deadBodyPlayerId = reader.ReadByte();
+                    foreach (var player in _game.Players.Where(p => p.Client.Id == deadBodyPlayerId))
```

Comment:
Instead of doing this, it would be more efficient to use `_game.GetClientPlayer(deadBodyPlayerId).Character`.